nuxeo-common/src/test/java/org/nuxeo/common/utils/TestTextTemplate.java

```diff
@@ -157,4 +158,32 @@ public void testEscapeVariable() {
         assertEquals("{$$CZjbsiX748UF583qkbinsQ==}", templates.processText("{$$CZjbsiX748UF583qkbinsQ==}"));
     }
 
+    @Test
+    public void testFreetemplateFileNameValidity() throws FileNotFoundException, IOException, TemplateException {
+        final TextTemplate textTemplate = new TextTemplate();
+        textTemplate.setFreemarkerParsingExtensions("nxftl");
+
+        final File testResouceDirectory = new File(getClass().getClassLoader().getResource("test-nxftl").getPath());
```

Comment:
Use `org.nuxeo.common.utils.FileUtils#getResourceFileFromContext`, used in other tests in nuxeo-common as well.

Reply:
I copied this from a few lines above, I had no idea about this.
I'll change all the occurrences in this file.